### ISSUE TITLE
Issue with SageMaker Deployment

### ISSUE THREAD

**Issue by NaxAlpha** [bentoml/aws-sagemaker-deploy#13]
**Describe the bug**

I am trying to deploy multiple versions (staging and production) of the same model on aws sagemaker using aws-sagemaker-deply tool. However after successfully deploying staging varient, I cannot deploy production version because of this error:
![image](https://user-images.githubusercontent.com/11090613/130733692-a51f0b28-256f-438e-93a4-1f5f5d628fd1.png)

**To Reproduce**

```bash
BENTO_BUNDLE_PATH=$(bentoml get Model:latest --print-location -q)
python deploy.py $BENTO_BUNDLE_PATH model-stg sagemaker_config.json
python deploy.py $BENTO_BUNDLE_PATH model-prd sagemaker_config.json  # <- error here
```

**Expected behavior**

It should have had successfully deployed but gives this error:

```
Export with name OutputApiId is already exported by stack model-stg-endpoint. Rollback requested by user.
```

**Screenshots/Logs**

N/A

**Environment:**
 - OS: Debian GNU/Linux 10 (buster)
 - Python Version: Python 3.7.10 | packaged by conda-forge
 - BentoML Version: bentoml, version 0.13.1

**Additional context**

**Comment by yubozhao** [comment]
@NaxAlpha This issue is fixed with the PR #9 . Feel free to reopen it, if you still are running into any issues.

**Comment by NaxAlpha** [comment]
Thanks. just tested it and it seems like it's working
